src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,47 @@
+@import "src/styles/_utils/_variables.scss";
+
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__wraper {
+    display: flex;
+    width: $box-size * 7;
+    flex-wrap: wrap;
+    gap: 1px; 
+  }
+  
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    width: $box-size;
+    height: $box-size;
+    border: solid black 1px;
+    box-sizing: border-box;
+    transition: 0.5s ease-in-out;
```

Comment:
don't use it like that, because it's the same as transition-duration: 0.5s
It's too hard for the browser
Check this
![image](https://user-images.githubusercontent.com/91411692/223077055-92cdea53-7d1e-4964-8773-2ab5364abfed.png)

also, create variable for transitionDuration
```suggestion
    transition: transform $transitionDuration ease-in-out, ...;
```